docs/source/user-guide/installation.md

```diff
@@ -28,9 +28,12 @@ Make sure the following requirements are met and software installed on your mach
 
 ## Requirements
 
-- [Apache Spark supported by Comet](overview.md#supported-apache-spark-versions)
```

Comment:
This was updated recently to link the other doc by @comphead in #975. I think it is correct, why changing it back?